app.py

```diff
@@ -9,13 +10,57 @@
 import requests
 import time
 import shlex
+import asyncio
+import aioslsk 
 from db import create_connection, create_table, insert_track, fetch_all_tracks, update_download_status
 from log_config import setup_logging
 from utils import sleep_interval
 from mutagen.mp3 import MP3
 from mutagen.id3 import ID3, TIT2, TPE1, TALB, TDRC
 from utils import sanitize_table_name
 
+#function to download a soulseek track
+async def download_from_soulseek(track_name, artist_name, download_path, username, password):
+    os.makedirs(download_path, exist_ok=True)
+
+    #creat a client
+    client = aioslsk.client()
+
+    try: 
+        #soulseek login 
+        await client.login(username, password)
+
+        #searching a file soulseek
+        search_query = f'{track_name} {artist_name}'
```

Comment:
In the original download_track function there was a treatment for the track and artist name, why was this removed here?

Reply:
mb, I will add character handling